wordpress-mobile/WordPress-iOS
# Draft implementation of the address wizard step

Implements #10327 

First draft implementation of the UI for the last step of the wizard: picking a site address.

In this PR:
- Implementation of the WebAddressStep
- Implementation of the WebAddressWizardContent
- Added a mock implementation of a service to fetch the addresses
- Added a property to the object tracking the data gathered through the wizard.
- Modified the way the headers are added to each step.
- Started applying styles.

Have in mind:
- The UI is still not final.
- There is a lot of duplication. The implementation of the UI of each of the steps is almost the same. I want to get into the "Site Info" step next, and after I have an idea of how the implementation of that step is going to look, I am sure we'll be able to extract most of the existing duplication.
- I haven't added any new tests yet. I will as soon as the implementation starts looking final.

To test:
- Run the tests
- Navigate to My Sites > Tap the + icon > Enhanced site creation
- Tap any row in the first screen. It should navigate to "What's the focus of your business?"
- Enter any text in the textfield and select any vertical
- The flow should navigate to "Choose a domain for your site"
- Enter any text in the textield. Selecting a domain will not have any effect
